releasenotes/notes/fix-initial-point-with-reps-4725be71d06cb747.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
```

Comment:
I think this is more deprecation section and a fixes section. That is what it would need to be if we are going to fix this on the existing code. And if we do it there its not needed here as such. We might want to give some clues in a migration guide but if we do a bug fix and the logic is already deprecated there then I guess to some extent you are already clued-in.

As to your comments, on the changes vis-a-vis deprecation, since this is "all new code" in this location it does not need it. But as I say we may need to give some hints on transitioning from current initial points to these new ones in migration guide.

Reply:
So, am I right in thinking that this PR does not require a releasenote, as this will be covered by the PR on the existing algorithms?

Reply:
It actually is still needed. Just that you should have two sections: `fixes` and `deprecates` (or whatever the default names are here).
In one you list the actual bug which is being fixed (which is the part that would also need to be backported to the stable branch later), and in the other section you list what you have `deprecated`. However, this part is arguably not needed in this part of the stack

Reply:
The release notes should only log changes relative to previous releases though, right? The initial points under `second_q` didn't exist in the previous release so we don't need to log the fix and nothing needs to be depreciated. The fix noted in the release note for PR #808 should cover the fix to the old initial points.

Reply:
Having the fix documented in the reno of the other PR seems fine. But I would still deprecate that the `excitation_list` cannot be set manually, simply because there is no other way the user would find out about this